Source/WebCore/Modules/highlight/HighlightRegister.cpp

```diff
@@ -74,7 +74,7 @@ void HighlightRegister::addAnnotationHighlightWithRange(Ref<StaticRange>&& value
     if (m_map.contains(annotationHighlightKey()))
         m_map.get(annotationHighlightKey())->addToSetLike(value);
     else
-        setFromMapLike(annotationHighlightKey(), Highlight::create(WTFMove(value)));
+        setFromMapLike(annotationHighlightKey(), Highlight::create({ WTFMove(value) }));
```

Comment:
This doesn't build on PS, but it's taking me a moment to figure out why, as it's quite hard to understand what the braces are precisely meant to refer to. Is the intention that everything but the initializer_list happens implicitly?

Reply:
Yes, the idea is that it would call the following constructor:
```
FixedVector(std::initializer_list<T> initializerList);
```
May need to use `FixedVector({ ... })` for PS or even `FixedVector<std::reference_wrapper<WebCore::AbstractRange>>({ ... })`.

Reply:
Hmm, you may be right, it may require parens _and_ braces.

Reply:
Hmm, I guess that may not be the source of the issue at all. The confusing part to me is how Ref becomes std::reference_wrapper, and indeed, it seems like that's what our compiler is stuck on.